test/chi/components/alert-toast.pug

```diff
@@ -1,19 +1,87 @@
 ---
 title: Alert Toasts
 ---
+- var sizes = ['h6', 'h5', 'h4', 'h3', 'h2', 'h1'];
```

Comment:
@jllr @davidojedacl Heading sizes should be defined at the root alert level (m-alert), not on a specific type of an alert (-toast). Also, can we remove all the documentation for this? We don't want to encourage developers to use every heading size in alerts, but if they do, we want to override the default margin so it's at least presentable. It's not something that requires this level of documentation.

Reply:
ok

Reply:
fixed